Python/libraries/recognizers-number/recognizers_number/number/spanish/extractors.py

```diff
@@ -62,24 +62,24 @@ def _extract_type(self) -> str:
 
     def __init__(self, placeholder: str = SpanishNumeric.PlaceHolderDefault):
         self.__regexes = [
-            ReVal(
-                re=SpanishNumeric.NumbersWithPlaceHolder(placeholder),
-                val='IntegerNum'),
-            ReVal(
-                re=SpanishNumeric.NumbersWithSuffix,
-                val='IntegerNum'),
-            ReVal(
-                re=self._generate_format_regex(LongFormatMode.INTEGER_DOT, placeholder),
-                val='IntegerNum'),
-            ReVal(
-                re=self._generate_format_regex(LongFormatMode.INTEGER_BLANK, placeholder),
-                val='IntegerNum'),
-            ReVal(
-                re=SpanishNumeric.RoundNumberIntegerRegexWithLocks,
-                val='IntegerNum'),
-            ReVal(
-                re=SpanishNumeric.NumbersWithDozenSuffix,
-                val='IntegerNum'),
+            # ReVal(
+            #     re=SpanishNumeric.NumbersWithPlaceHolder(placeholder),
+            #     val='IntegerNum'),
+            # ReVal(
+            #     re=SpanishNumeric.NumbersWithSuffix,
+            #     val='IntegerNum'),
+            # ReVal(
+            #     re=self._generate_format_regex(LongFormatMode.INTEGER_DOT, placeholder),
+            #     val='IntegerNum'),
+            # ReVal(
+            #     re=self._generate_format_regex(LongFormatMode.INTEGER_BLANK, placeholder),
+            #     val='IntegerNum'),
+            # ReVal(
+            #     re=SpanishNumeric.RoundNumberIntegerRegexWithLocks,
+            #     val='IntegerNum'),
+            # ReVal(
+            #     re=SpanishNumeric.NumbersWithDozenSuffix,
+            #     val='IntegerNum'),
             ReVal(
                 re=SpanishNumeric.AllIntRegexWithLocks,
                 val='IntegerSpa'),

```

Comment:
What's this regex doing? (I have no idea)

Reply:
Me netiher, I deactivated other regexs relative to work with digits. I think that this regex works with text to integer but i don't know what kind of number transcript exactly.

Reply:
Confirmed, this is the main regex to transcribe text cardinal numbers to digits